docs/user.md

```diff
@@ -662,4 +667,4 @@ since Teflo does not have similar functionality. Where we suggest users should s
 away from Linchpin hooks is if any `postup` hooks have been configured. That is where thos scripts/playbooks
 can be moved over and configured in Teflo's 
 [Orchestrate](https://docs.engineering.redhat.com/display/CentralCI/Orchestrate) section. It can help 
```

Comment:
Need to change this link to the correct one from readthedocs. This is pointing to the older docs on confluence
https://teflo.readthedocs.io/en/latest/users/definitions/orchestrate.html

Reply:
done